tests/pytests/common.py

```diff
@@ -857,7 +857,11 @@ def set_tight_maxmemory_for_oom(env, memory_limit_per = 0.8):
     # Get current memory consumption value
     memory_usage = env.cmd('INFO', 'MEMORY')['used_memory']
     # Set memory limit to less then memory limit
-    env.expect('config', 'set', 'maxmemory', int(memory_usage*(1/(memory_limit_per-0.01)))).ok()
+    required_memory = memory_usage * (1/memory_limit_per)
+    # Round up and add 1
+    new_memory = int(required_memory) + bool(required_memory%1) + 1
```

Comment:
why not use `round`?

Reply:
round rounds to nearest rather then up,
I can change to round and add 2 it's not that important

Reply:
[math.ceil](https://docs.python.org/3/library/math.html#math.ceil) rounds up. But I agree this is not very important